packages/autorest.typescript/test/rlcIntegration/dpgCustomizationRest.spec.ts

```diff
@@ -44,7 +44,7 @@ describe("DPGCustomization Client", () => {
       const initialResponse = await client
         .path("/customization/lro/{mode}", "raw")
         .put();
-      const poller = getLongRunningPoller(client, initialResponse);
+      const poller = await getLongRunningPoller(client, initialResponse);
```

Comment:
why is this not a breaking ?

Reply:
I wonder if the changes made in RLC pollingHelper are also behind the use-legacy-lro flag ?

Reply:
Good point, addressed in https://github.com/Azure/autorest.typescript/pull/1586/commits/49a3d53dc802dbb6a7ad1ae29fcb85ca15909512.